### ISSUE TITLE
NTR: Plantar creases absent at birth (Skin and mucosa)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8012]
New term request Plantar creases absent at birth (elthat_160819045318): Planter  UMLS: 2081659

**Comment by pnrobinson** [comment]
@MickeySegal
what is the context?
Probably it would be better not to have the term (just "Absent plantar creases") include the onset, this can be added separately?

**Comment by MickeySegal** [comment]
We changed this to "Plantar creases decreased".  Seen in 
Renal hypodysplasia/aplasia 4, GFRA1-related
Apert syndrome
Atelis syndrome 2, SMC5-related

**Comment by pnrobinson** [comment]
In OMIM, Atelis 2 is described as Single plantar creases.
I cannot find a reference to Apert.
For Renal hypodysplasia/aplasia 4, and article says "reduced plantar creases" but there is no photograph and the sole of the foot seems to have less pronounced creases that the palm, so I am not sure this is very convincing.

**Comment by MickeySegal** [comment]
Woodward PJ et al. (2016) Diagnostic Imaging: Obstetrics
"Clinical photograph of the plantar surface of the foot of a stillborn with Apert syndrome reveals complete syndactyly of the toes as well as the apparent absence of normal plantar creases."

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001252
- label: Reduced plantar creases
- parents: [Abnormality of the plantar skin of foot (HP:0100872)](https://hpo.jax.org/app/browse/term/HP:0100872)
- pmid: PMID:6869501
- def: The creases (wrinkles) on the soles of the feet are less pronounced than normal, or absent.
- comment: This finding may be seen in premature infants and in some conditions such as GFRA1-related Apert syndrome, renal hypodysplasia/aplasia 4, and Atelis syndrome 2.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8012
- synonyms:
